docs/drupal/first-deployment-of-drupal.md

```diff
@@ -124,7 +124,7 @@ Do you really want to continue? (y/n):
 
 The reason for that is that the Drupal cannot resolve the path of the files directory. This most probably has to do that the Drupal is not fully configured or has a missing database. For a workaround you can use `drush rsync @self:sites/default/files @develop:sites/default/files`, but we suggest that you actually check your local and remote Drupal \(you can test with `drush status` to see if the files directory is correctly configured\).
 
-## 7. It's done!
+## 7. It's done
```

Comment:
Is there a way to tell the linter to ignore things? I know it's little, but i care about the tone of our docs and i want the occasional exclamation point in my headers.

Reply:
Yeah: https://nvuillam.github.io/mega-linter/descriptors/markdown_markdownlint/#readme - I used the defaults